selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/CreateWorkspaceFromEclipseCheStackTest.java

```diff
@@ -0,0 +1,143 @@
+import static org.eclipse.che.selenium.core.constant.TestProjectExplorerContextMenuConstants.ContextMenuCommandGoals.BUILD_GOAL;
+import static org.eclipse.che.selenium.core.constant.TestProjectExplorerContextMenuConstants.ContextMenuCommandGoals.DEBUG_GOAL;
+import static org.eclipse.che.selenium.core.constant.TestProjectExplorerContextMenuConstants.ContextMenuCommandGoals.RUN_GOAL;
+import static org.eclipse.che.selenium.pageobject.dashboard.NewWorkspace.Stack.ECLIPSE_CHE;
+import static org.openqa.selenium.Keys.ENTER;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Inject;
+import java.util.List;
+import org.eclipse.che.selenium.core.client.TestWorkspaceServiceClient;
+import org.eclipse.che.selenium.core.user.DefaultTestUser;
+import org.eclipse.che.selenium.pageobject.CheTerminal;
+import org.eclipse.che.selenium.pageobject.Consoles;
+import org.eclipse.che.selenium.pageobject.Ide;
+import org.eclipse.che.selenium.pageobject.ProjectExplorer;
+import org.eclipse.che.selenium.pageobject.dashboard.CreateWorkspaceHelper;
+import org.eclipse.che.selenium.pageobject.dashboard.Dashboard;
+import org.openqa.selenium.By;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+/** @author Skoryk Serhii */
+public class CreateWorkspaceFromEclipseCheStackTest {
+
+  private static final String WORKSPACE_NAME = generate("workspace", 4);
+  private static final String CONSOLE_JAVA_PROJECT = "console-java-simple";
+  private static final String WEB_JAVA_SPRING_PROJECT = "web-java-spring";
+
+  private List<String> projects = ImmutableList.of(CONSOLE_JAVA_PROJECT, WEB_JAVA_SPRING_PROJECT);
+  private By textOnPreviewPage = By.xpath("//span[text()='Enter your name: ']");
+
+  @Inject private Ide ide;
+  @Inject private Consoles consoles;
+  @Inject private Dashboard dashboard;
+  @Inject private CheTerminal terminal;
+  @Inject private DefaultTestUser defaultTestUser;
+  @Inject private ProjectExplorer projectExplorer;
+  @Inject private CreateWorkspaceHelper createWorkspaceHelper;
+  @Inject private TestWorkspaceServiceClient workspaceServiceClient;
+
+  @BeforeClass
+  public void setUp() {
+
+    dashboard.open();
+  }
+
+  @AfterClass
+  public void tearDown() throws Exception {
+    workspaceServiceClient.delete(WORKSPACE_NAME, defaultTestUser.getName());
+  }
+
+  @Test
+  public void checkWorkspaceCreationFromJavaStack() {
+    createWorkspaceHelper.createWorkspaceFromStackWithProjects(
+        ECLIPSE_CHE, WORKSPACE_NAME, projects);
+
+    ide.switchToIdeAndWaitWorkspaceIsReadyToUse();
+
+    projectExplorer.waitProjectInitialization(CONSOLE_JAVA_PROJECT);
+    projectExplorer.waitProjectInitialization(WEB_JAVA_SPRING_PROJECT);
+  }
+
+  @Test(priority = 1)
+  public void checkConsoleJavaSimpleProjectCommands() {
+    consoles.executeCommandFromProjectExplorer(
```

Comment:
It is not enough to check Eclipse Che stack with Java Simple Project. This stack is dedicated to work develop something for Eclipse Che project including Eclipse Che extensions, and Eclipse Che project itself, and there is separate issue for this staff https://github.com/eclipse/che/issues/10107, which becomes partly resolved.